### PR TITLE
New post - aalvz

### PR DESCRIPTION
**Pull request by AAlvz**
New post ready for revision

### CODE REVIEW

File: 2016/07/about-tinkerware.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+This "small" step is the difference between (hopefully) *creating a great
+service used by a lot of people* and  - forgetting about that cool small project
+you once did in the past...
+
+Other times, paying third parties to perform all these tasks is not an option
+since it gets expensive, so why not making these configurations easy for you
+and let you handle your own server with freedom with the posibility of growth?
+
+*You can achieve this* with *environments*.
```
Comment (iLeonelPerea) [inline 2016/07/about-tinkerware.md]: U need to type *\* text *\* to make some text bold.

File: 2016/07/about-tinkerware.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  - Configure your server
+  - Secure your server
+  - Deploy your code
+  - Keep an eye on your server
+  - Hope that nothing will break.
+
+...a lot of times it's hard to find the time to do all this.
+
+This "small" step is the difference between (hopefully) *creating a great
+service used by a lot of people* and  - forgetting about that cool small project
```
Comment (iLeonelPerea) [inline 2016/07/about-tinkerware.md]: two **

**Comment by AAlvz** [comment]
@iLeonelPerea like that? =)

**Comment by iLeonelPerea** [comment]
http://lgtm.in/p/MKL5KRvKn

**Comment by iLeonelPerea** [comment]
image

**Pull request merged**